uq360/algorithms/blackbox_metamodel/confidence_predictor.py

```diff
@@ -0,0 +1,45 @@
+
+from uq360.algorithms.blackbox_metamodel.predictors.predictor_driver import PredictorDriver
+from uq360.algorithms.posthocuq import PostHocUQ
+
+
+"""PostHocUQ model based on the "confidence" performance predictor 
```

Comment:
I think docstrings should be with `__init__()` for the apidocs to be populated correctly. 

Instructions for checking api-docs https://github.com/IBM/UQ360/tree/main/docs